.circleci/config.yml

```diff
@@ -7,12 +7,14 @@ jobs:
     steps:
       - checkout
       - restore_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "package.json" }}
+          keys:
+            - deps2-{{ checksum "package.json" }}
+            - deps2-
       - run: npm install
       - save_cache:
-          key: deps1-{{ .Branch }}-{{ checksum "package.json" }}
+          key: deps2-{{ checksum "package.json" }}
           paths:
-            - node_modules
+            - ~/.npm
```

Comment:
I'm curious why this is preferred over caching the whole node_modules dir?

Reply:
In going back to get the link to my source for this, I think I misread it. I'll change this back.